src/components/ErrorComponents/DefaultErrorComponent.test.tsx

```
@@ -68,6 +69,62 @@ describe('DefaultErrorComponent', () => {
 
+  describe('Sentry error reporting', () => {
+    it('reports the original error to Sentry with react componentStack and extra context', async () => {
+      const error = new Error('router context is null');
+      const componentStack = '\n    in NavLink\n    in SegmentProvider\n    in ScalprumRoot';
+
+      renderWithProviders(<DefaultErrorComponent error={error} errorInfo={{ componentStack }} />);
+
+      await waitFor(() => {
+        expect(Sentry.captureException).toHaveBeenCalledWith(
+          error,
+          {
```

ESLint

[error] Replace `⏎··········error,⏎·········` with `error,`

(prettier/prettier)